test/samples/solidity/consts/consts.sol

```diff
@@ -0,0 +1,22 @@
+import "./lib_a.sol";
+import "./lib_a.sol" as LibA;
+
+uint constant SOME_CONST = 100;
+uint constant SOME_OTHER = 15;
+uint constant SOME_ELSE = SOME_CONST + SOME_OTHER;
+uint constant C2 = SOME_ELSE + ANOTHER_CONST;
+uint constant C3 = SOME_ELSE + LibA.ANOTHER_CONST;
+uint constant C4 = -SOME_CONST;
+bool constant C5 = false;
+uint constant C6 = C5 ? SOME_ELSE : C3;
+uint constant C7 = LibA.ANOTHER_CONST + LibB.AND_ANOTHER_CONST;
+// uint constant C8 = LibA.ANOTHER_CONST + LibA.LibB.AND_ANOTHER_CONST;
```

Comment:
@cd1m0

There is a bug in `InferType` here, that does not allow member access to be resolved to a `ModuleType`. Also `ModuleType` is an intenral node, that is used only by a typeString parser for test purposes. I would need a help to fix it.

Still, it is not pressing, as such language construct appears rarely and modern linters are encouraging to use specific imports instead of aliasing whole modules.